Doc/whatsnew/3.7.rst

```diff
@@ -865,143 +1667,245 @@ argument to generate a compressed archive.  A command line option
 (Contributed by Zhiming Wang in :issue:`31638`.)
 
 
+zipfile
+-------
+
+:class:`~zipfile.ZipFile` now accepts the new *compresslevel* parameter to
+control the compression level.
+(Contributed by Stephen J. Turnbull. in :issue:`28102`.)
```

Comment:
This references the incorrect issue - it was actually [21417](https://bugs.python.org/issue21417), and the attribution is also scrambled.

Reply:
Fixed. Thanks for catching this.